Update scripts for Matlab R2022a and Vivado 2022.2

StancaPop opened the pull request:
Update the existing scripts to work with Matlab R2022a and Vivado 2022.2.

It has been successfully tested along with the Precision Toolbox.

StancaPop committed:
Update scripts for Matlab R2022a and Vivado 2022.2

tfcollins commented on build_bsp.sh:
```diff
@@ -72,75 +67,34 @@ do
+cp scripts/adi_build.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/adi_build.tcl
+
+# Copy fsbl files
+cp scripts/fsbl_build_zynq.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fsbl_build_zynq.tcl
+cp scripts/fsbl_build_zynqmp.tcl ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fsbl_build_zynqmp.tcl
+cp scripts/pmufw_zynqmp.tcl  ../hdl/vendor/AnalogDevices/vivado/projects/scripts/pmufw_zynqmp.tcl
+cp scripts/fixmake.sh  ../hdl/vendor/AnalogDevices/vivado/projects/scripts/fixmake.sh
```
Not necessarily anymore if the adi_build.tcl script ignore them

StancaPop replied:
Deleted the unnecessary files.

tfcollins commented on fixmake.sh:
```diff
@@ -0,0 +1,3 @@
+#grep "CC_FLAGS :=" pmufw/Makefile | grep -e "-Os" || sed -i '/-mxl-soft-mul/ s/$/ -Os -flto -ffat-lto-objects/' pmufw/Makefile
+cd pmufw
+make
```
Can remove this file. Not necessary anymore

StancaPop replied:
Removed.

tfcollins reviewed (changes requested):
Need some minor cleanup

StancaPop committed:
build_bsp.sh: Remove redundant scripts dependancies

StancaPop committed:
Fix BOOT.BIN generation

StancaPop commented:
Made the required cleanup. Fixed the BOOT.BIN generation and added the necessary boot files. Tested along with Precision Toolbox within the add_cn0585_fmcz branch.
